modules/assisted-installer-how-to-use-the-assisted-installer.adoc

```diff
@@ -0,0 +1,44 @@
+// This is included in the following assemblies:
+//
+// installing-on-prem-assisted.adoc
+:_content-type: CONCEPT
+
+[id="how-to-use-the-assisted-installer_{context}"]
+= How to use the Assisted Installer
```

Comment:
This doc isn't really about "how" to use the Assisted Installer, more like when or why.  Maybe just "Using the Assisted Installer"?